be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/SocialMediaHandler.scala

```diff
@@ -93,16 +86,24 @@ class SocialMediaHandler(dbRef: => AskableActorRef) extends MessageHandler {
   }
 
   def handleAddReaction(rpcMessage: JsonRpcRequest): GraphMessage = {
-    writeAndPropagate(rpcMessage)
+    val ask: Future[GraphMessage] = dbAskWritePropagate(rpcMessage)
```

Comment:
Here, I'm not sure whether it is more relevant to keep the Awaits explicit or not tbh